books/RayTracingTheRestOfYourLife.html

```diff
@@ -357,14 +357,15 @@
         }
         ...
         ray get_ray(int i, int j, int s_i, int s_j) const {
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ C++ highlight
             // Get a randomly-sampled camera ray for the pixel at location i,j, originating from
```

Comment:
This isn't directly related to this PR but the "randomly sampled" part here feels a little redundant, how about rephrasing this sentence to something like:

```
// Build a camera ray originating from the camera defocus disk and directed at
// a randomly sampled point around the pixel at location i,j.
```

Reply:
Will do.